bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,238 @@
+		</channels>
+
+		<config-description-ref uri="thing-type:boschshc:device"/>
+
+	</thing-type>
+
+	<thing-type id="thermostat">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="shc"/>
+		</supported-bridge-type-refs>
+
+		<label>Thermostat</label>
+		<description>Bosch Thermostat</description>
+
+		<channels>
+			<channel id="temperature" typeId="temperature"/>
+			<channel id="valve-tappet-position" typeId="valve-tappet-position"/>
+		</channels>
+
+		<config-description-ref uri="thing-type:boschshc:device"/>
+
+	</thing-type>
+
+	<thing-type id="climate-control">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="shc"/>
+		</supported-bridge-type-refs>
+
+		<label>Climate Control</label>
+		<description>Bosch Climate Control. This is a virtual device which is automatically created for all rooms that have
+			thermostats in it.</description>
+
+		<channels>
+			<channel id="temperature" typeId="temperature"/>
+			<channel id="setpoint-temperature" typeId="setpoint-temperature"/>
+		</channels>
+
+		<config-description-ref uri="thing-type:boschshc:device"/>
+
+	</thing-type>
+
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Current measured temperature.</description>
+		<state min="0" max="40" step="0.5" pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="temperature-rating">
+		<item-type>String</item-type>
+		<label>Temperature Rating</label>
+		<description>Rating of the currently measured temperature.</description>
+		<state readOnly="true">
+			<options>
+				<option value="GOOD">Good Temperature</option>
+				<option value="MEDIUM">Medium Temperature</option>
+				<option value="BAD">Bad Temperature</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="humidity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<description>Current measured humidity.</description>
+	</channel-type>
+
+	<channel-type id="humidity-rating">
+		<item-type>String</item-type>
+		<label>Humidity Rating</label>
+		<description>Rating of current measured humidity.</description>
+		<state readOnly="true">
+			<options>
+				<option value="GOOD">Good Humidity</option>
+				<option value="MEDIUM">Medium Humidity</option>
+				<option value="BAD">Bad Humidity</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="energy-consumption">
+		<item-type>Number:Energy</item-type>
+		<label>Energy consumption (Wh)</label>
+		<description>Energy consumption of the device.</description>
+	</channel-type>
+
+	<channel-type id="power-consumption">
+		<item-type>Number:Power</item-type>
+		<label>Power consumption (W)</label>
+		<description>Current power consumption of the device.</description>
+	</channel-type>
+
+	<channel-type id="purity">
+		<item-type>Number</item-type>
+		<label>Purity</label>
+		<description>Purity of the air.</description>
+	</channel-type>
```

Comment:
Is this supposed to be writable? Also please add some details to the README as to what purity values are supposed to mean. Is a higher value better or worse?

Reply:
@stefan-kaestle Maybe you have an information about the purity value? I can not find any further description in the Bosch docs.

Reply:
Flagged the channels that are read-only

Reply:
Alright, I received some information from the Bosch devs: https://github.com/BoschSmartHome/bosch-shc-api-docs/issues/40 Added them to the channel definition and the Readme.md